Assets/MixedRealityToolkit/Utilities/Gltf/Serialization/GltfChunkType.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace Microsoft.MixedReality.Toolkit.Utilities.Gltf.Serialization
+{
+    public enum GltfChunkType : uint
+    {
+        Json = 0x4e4f534a,
```

Comment:
Similar to schema enums, this seems like another location where a summary call out for the value could help troubleshoot in any future regressions

Reply:
Defined in the schema https://github.com/KhronosGroup/glTF/tree/master/specification/2.0/schema